### PR TITLE
BC001 TE01 - first draft

### PR DESCRIPTION
**Pull request by pjanck**

### CODE REVIEW

File: tests/TE01/readme.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+- All following rules:
+  - ??
+
+> TODO *remains to be determined*
+
+#### Test case-specific checks
+
+Link to IDS file: `[ABCD123.ids]()` :construction:
+
+- There must be 1 instance of `IfcTriangulatedIrregularNetwork` in the dataset
```
Comment (BenzclyZhang) [inline tests/TE01/readme.md]: Number of IfcGeographicElement?

File: tests/TE01/readme.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+There should be four triangulated irregular networks (TINs) defined as described in the [Dataset description](Dataset/README.md).
+
+
+
+### Control parameters
+
+The following steps should be performed in order to corroborate that the software is working as expected.
+
+After importing the reference file (`BC001_TE01_reference.ifc`):
+
```
Comment (BenzclyZhang) [inline tests/TE01/readme.md:98]: Coordinates of 4 corners

File: tests/TE01/readme.md (hunk 1 above)
Comment (BenzclyZhang) [reply tests/TE01/readme.md]: Relation to IfcSite?

Comment (pjanck) [reply tests/TE01/readme.md]: Done in 05dbda352bac9ed9afc48bd7b8424e931e3ab551

File: tests/TE01/readme.md (hunk 2 above)
Comment (pjanck) [reply tests/TE01/readme.md:98]: Done in 5ac09bb5cd7dd466d9077e51bd623610cf70cedf

**Pull request closed without merging**